app/views/passengers/brochure/_scheduling.html.haml

```diff
@@ -9,7 +9,7 @@
     on Friday morning to set up your rides for the following week.
   %li
     #{t 'department.name'} vans travel on campus and in close proximity to the
-    University, we do not travel more than 3 miles from Du Bois Library.
+    University, we do not travel more than 3 miles away or more than 10 minutes from the Du Bois Library.
```

Comment:
This technically fits our style guide, but I've been keeping to 80 characters in the brochure because it's so prose-like.

```suggestion
    University, we do not travel more than 3 miles away or more than 10 minutes
    from the Du Bois Library.
```